docs/MultiTableDatabase-batchsource.md

```diff
@@ -62,45 +63,59 @@ This example reads from all tables in the 'customers' database on host 'host123.
         "type": "batchsource",
         "properties": {
             "connectionString": "jdbc:mysql://host123.example.net/customers",
-            "jdbcPluginName": "mysql"
+            "jdbcPluginName": "mysql",
+            "numSplit": "2"
         }
     }
 
 Suppose you have two tables in the 'customers' database. The first table is named 'accounts' and contains:
 
     +-----+----------+------------------+
-    | id  | name     | email            |
+    | ID  | name     | email            |
     +-----+----------+------------------+
     | 0   | Samuel   | sjax@example.net |
     | 1   | Alice    | a@example.net    |
+    | 2   | Bob      | b@example.net    |
+    | 3   | John     | j@example.net    |
     +-----+----------+------------------+
 
 The second is named 'activity' and contains:
 
-    +--------+----------+--------+
-    | userid | item     | action |
-    +--------+----------+--------+
-    | 0      | shirt123 | view   |
-    | 0      | carxyz   | view   |
-    | 0      | shirt123 | buy    |
-    | 0      | coffee   | view   |
-    | 1      | cola     | buy    |
-    +--------+----------+--------+
+    +-----+--------+----------+--------+
+    | ID  | userid | item     | action |
+    +-----+--------+----------+--------+
+    | 0   | 0      | shirt123 | view   |
+    | 1   | 0      | carxyz   | view   |
+    | 2   | 0      | shirt123 | buy    |
+    | 3   | 0      | coffee   | view   |
+    | 4   | 1      | cola     | buy    |
+    | 5   | 1      | pepsi    | buy    |
+    +-----+--------+----------+--------+
+
+You will have 4 splits (2 per each table) with such queries:
```

Comment:
need to mention that the ID column is the primary key for both tables.

Reply:
done